R/write.R

```diff
@@ -5,139 +5,205 @@
 #' exists in a separate sheet.
 #'
 #' @param supertbl A supertibble generated using `read_redcap()`
-#' @param labelled Whether or not to include labelled outputs, default `FALSE`.
-#' Requires use of `make_labelled()`.
-#' @param labelled_sheets Whether or not to include labels in the XLSX sheets
-#' instead of raw values. Default `FALSE`.
 #' @param file A character string naming an xlsx file
-#' @param incl_supertbl Include a sheet capturing the supertibble output.
+#' @param labelled Whether or not to include labelled outputs, default `NULL`.
+#' Requires use of `make_labelled()`.
+#' @param use_labels_for_sheet_names Whether or not to include labels in the XLSX sheets
+#' instead of raw values. Default `TRUE`.
+#' @param include_toc_from_supertbl Include a sheet capturing the supertibble output.
 #' Default `TRUE`.
-#' @param incl_meta Include a sheet capturing the combined output of the
+#' @param include_metadata Include a sheet capturing the combined output of the
 #' supertibble `redcap_metadata`. Default `TRUE`.
-#' @param tableStyle Any excel table style name or "none" (see "formatting"
-#' vignette in \link[openxlsx]{writeDataTable}). Default "TableStyleLight10".
+#' @param table_style Any excel table style name or "none" (see "formatting"
+#' vignette in \link[openxlsx2]{wb_add_data_table}). Default "tableStyleLight8".
+#' @param set_col_widths Width to set columns across the workbook. Default
+#' "auto", otherwise a numeric value. Standard Excel is 8.43.
+#' @param recode_yn Convert REDCap yesno questions from TRUE/FALSE to "yes"/"no"
+#' for readability. Default TRUE.
+#' @param na_strings Value used for replacing NA values from x. Default
+#' `na_strings()` uses the special ⁠#N/A⁠ value within the workbook.
 #'
```

Comment:
I think we could add an `overwrite` param. Can default to `TRUE`. Thoughts?

Reply:
Looks like this was resolved